app/models/user.rb

```diff
@@ -21,5 +22,12 @@ def self.find_for_google_oauth2(access_token, signed_in_resource=nil)
   def remember_me
     true
   end
-
+  def self.reminder_task
+    User.all.each do |user|
+      if TodoList.joins(:users).where('users.id' => user.id).
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.